css/style.css

```diff
@@ -641,20 +652,468 @@ hr {
   font-weight: 400;
 }
 
+.footer {
+  position: absolute;
+  width: 100%;
+  height: 72px;
+  top: 6738px;
+}
+
 #c-line {
   height: 1px;
   margin-top: 120px;
   align-items: center;
-  margin-left: 7px;
+  margin-left: 0;
   color: white;
   background-color: #dfe1e6;
 }
 
 #s-icons {
   margin-top: 15px;
-  margin-left: 120px;
+  margin-left: 85px;
 }
 
 .l-line {
-  margin-left: 35px;
+  margin-left: 0;
+}
+
+@media (min-width: 768px) {
+  body {
+    width: wider;
+    height: 72px;
+  }
+
+  nav {
+    color: #fff;
+    margin: 2% 2%;
+  }
+
+  /* .header {
+    width: 1440px;
+    height: 72px;
+  } */
+
+  .fa {
+    display: none;
+  }
+
+  .hidden {
+    position: absolute;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 50px;
+    right: 100px;
+    top: calc(6% - 40px / 2);
+  }
+
+  .hidden div {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+  }
+
+  .hidden a {
+    text-decoration: none;
+    color: #fff;
+  }
+
```

Comment:
- [ ] Your navigation links are aligned well 👏. However, I would suggest you add the correct `font` to them which matches the one in the Figma design you have chosen. Look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/177883323-e2795bc0-6008-499e-a573-a429bd9d4e13.png)

**Figma design**
![image](https://user-images.githubusercontent.com/90222110/177883530-bebe001a-9c0c-4aae-b847-519348538d76.png)
You can see that they look different **in terms of the font** so please correct on that 🏃‍♂️ 
💡 **HINT** : I think they have used `font-family: Inter, sans-serif`. Check the Figma design again to confirm it and then use it 😃